src/Core/Configurations/JsonConfigSchemaValidator.cs

```diff
@@ -28,6 +29,19 @@ public JsonConfigSchemaValidator(ILogger<JsonConfigSchemaValidator> jsonSchemaVa
         _fileSystem = fileSystem;
     }
 
+    /// <summary> 
+    /// Sets the logger, file system, and HttpClient for the JSON config schema validator. 
+    /// </summary> 
+    /// <param name="jsonSchemaValidatorLogger">The logger to use for the JSON schema validator.</param> 
```

Comment:
Note in PR description why we need second constructor versus just adding this new param to the original constructor.
backwards compat?

Reply:
If you use constructor without httpClient, won't the call to `GetJsonSchema` fail because client is null.

Reply:
client is never null. it is initialized in the begining itself. But good point. I can actually have just one constructor as well.